terraform/environment/credentials.tf

```
@@ -28,4 +32,14 @@ provider "aws" {
   }
 }
 
+provider "aws" {
+  region = "eu-west-1"
+  alias  = "management"
+
+  assume_role {
+    role_arn     = "arn:aws:iam::311462405659:role/${var.management_role}"
+    session_name = "terraform-session"
```

I don't think this is being done elsewhere but I think it's worth naming the session to show that it's for opg-metrics. The management account is used by pretty much every service/ws account, so it will be helpful to differentiate for audit.

```suggestion
    session_name = "opg-metrics-terraform-session"
```